opentelemetry-etw-metrics/src/exporter/mod.rs

```diff
@@ -70,14 +70,22 @@ impl PushMetricsExporter for MetricsExporter {
             .encode(&mut byte_array)
             .map_err(|err| MetricsError::Other(err.to_string()))?;
 
-        let result = etw::write(&byte_array);
-        if result != 0 {
+        if (byte_array.len()) > etw::MAX_EVENT_SIZE {
             global::handle_error(MetricsError::Other(format!(
-                "Failed to write ETW event with error code: {}",
-                result
+                "Exporting failed due to event size {} exceeding the maximum size of {} bytes",
+                byte_array.len(),
+                etw::MAX_EVENT_SIZE
             )));
+        } else {
+            let result = etw::write(&byte_array);
+            // println!("Exported {} bytes to ETW", byte_array.len());
```

Comment:
Nit - remove the comment?

Reply:
I am really looking to get this https://github.com/open-telemetry/opentelemetry-rust/issues/1146 resolved, so we can liberally do internally logging. At this point, we are limited to logging in Error situations. (So this was my layman's attempt to figure out the serialized byte size, in various scenarios.!)

Reply:
I think it would be better to add a TODO statement here instead of having the `println!` comment in that case.